Objects/tupleobject.c

```diff
@@ -38,10 +38,13 @@ Py_ssize_t _Py_tuple_zero_allocs;
    many of them will be dead before. Therefore, a tracking rate close to 100%
    does not necessarily prove that the heuristic is inefficient.
 */
+#define TRACK_TUPLE(op) (_PyObject_GC_TRACK(op))
```

Comment:
> I suggest to rename to _PyTuple_GC_TRACK().

I'm not sure about `_PyTuple` prefix, since I had impression that it's for more public stuff, but this macro is completely internal.

Reply:
This is no strict rules for names. IMHO TRACK_TUPLE() name is too generic. Call it TUPLE_GC_TRACK() if you prefer.